server/middlewares/validators/videos/videos.ts

```diff
@@ -239,7 +239,7 @@ const videosChangeOwnershipValidator = [
     const nextOwner = await AccountModel.loadLocalByName(req.body.username)
     if (!nextOwner) {
       res.status(400)
-        .type('json')
+        .json({ error: 'Changing video ownership to a remote account is not supported yet' })
         .end()
```

Comment:
Since you use `res.json()`, there is no need anymore for the `res.end()`, which is typically only used if we want to end the response without providing any data.

Reply:
Hi, thanks for the info, I will change that. But in the video.ts file all the validators build their error response in that way. Should I refactor theses also ?

Reply:
If you have time sure! :+1:

Reply:
If you don't mind that would be great :)